### ISSUE TITLE
If there is a windicss.config.ts file in the project, the plug-in will be invalid

### ISSUE THREAD

**Issue by jiangmaniu** [windicss/windicss-intellisense#149]
https://user-images.githubusercontent.com/50100681/119477863-ec6e1400-bd81-11eb-9e3a-b3818e44c8f9.mp4

If there is a `windicss.config.ts` file in the project, and `Loading Config File: xxx` text (as shown in the picture below) does not appear in the `OUTPUT`, it seems to indicate that the loading is not successful and Windi Css Intellisense will not work.

Under what circumstances would the windicss.config.ts file be loaded as shown below to make the plug-in work?

![`4T~G P1P56AH7CHS)@0ME2](https://user-images.githubusercontent.com/50100681/119478294-47a00680-bd82-11eb-96f2-eddba24e7eeb.png)

I tried to find out the problem, but unfortunately I couldn't find it. This problem has been bothering me and prevented me from developing efficiently through the plug-in in the project.😢

**Comment by alexanderniebuhr** [comment]
@jiangmaniu I can't reproduce it. Can you share a repo?

**Comment by alexanderniebuhr** [comment]
one thing I noticed, that VS Code has blue curly underline on `vite-plugin-windicss`. Are you sure it is installed with `npm i`?

**Comment by jiangmaniu** [comment]
Of course, I have installed the `vite-plugin-windicss` dependency package

![image](https://user-images.githubusercontent.com/50100681/119482917-ff371780-bd86-11eb-9d58-a4a44d09ef2e.png)

The blue curve is because I installed the `Code Spell Checker` plug-in in `vscode` to check for incorrect English words (I am trying to learn English, but the current English level is not very good. The communication with you is through Google Translation. I am afraid that there will be low-level coding errors that English words are not fighting for, so I need this plugin to help me correct it.)

I tried to disable all `vs code` plug-ins and it worked well. The `OUTPUT` window at the bottom also showed that the configuration file was loaded.

![image](https://user-images.githubusercontent.com/50100681/119484241-66090080-bd88-11eb-8a1f-bc5e75ecb5b4.png)

So I think it is the conflict of `vs code` plug-in that caused this problem. When I searched Issuse, I found that #132  proposed the `Auto Rename Tag` plug-in will cause conflicts, but I tried it and it didn’t work for me. I disabled it. The plug-in still has this problem. So I think the conflicts caused by other `vs code` plugins. I am disabling or enabling the installed plugins one by one. I want to find this conflicting plugin, but unfortunately, I have been searching for a day and haven’t found any relevant Conflicting plugin.

**Comment by alexanderniebuhr** [comment]
please give me a list of plugins you use. screenshot is ok as well!

**Comment by alexanderniebuhr** [comment]
And if you can confirm it only breaks with `.ts`? `.js` files work fine?

**Comment by jiangmaniu** [comment]
My project uses `TypeScript` so I only tested `.ts`. I just tested the `.js` file, and it seems that the plugin will not work.

<details>
  <summary>The following is a picture of the list of plugins installed by my `vs code`</summary>
  
![img](https://user-images.githubusercontent.com/50100681/119490187-24c81f00-bd8f-11eb-8284-5085f657171b.png)

</details>

**Comment by alexanderniebuhr** [comment]
thanks. we are checking

**Comment by HenryC-3** [comment]
Hi, @alexanderniebuhr @jiangmaniu , I met the same problem.

it can be reproduced in `vite + windicss` or `vuecli + windicss` project by following steps

1. [windicss-vite-html-demo](https://github.com/HenryC-3/windicss-vite-demo)
2. disable all plugins in the workspace except `WindiCSS IntelliSense` and `Auto Rename Tag` plugin
3. reload window (IntelliSense works fine)
4. modify `windi.config.ts` (IntelliSense doesn't work)

I fixed this problem by disabling [Auto Rename Tag ](https://marketplace.visualstudio.com/items?itemName=formulahendry.auto-rename-tag) plugin, then reloading the vscode window

https://user-images.githubusercontent.com/41246463/119538200-738cad80-bdbd-11eb-9946-8e639360c1af.mov

**Comment by alexanderniebuhr** [comment]
@HenryC-3 thanks for letting us know. But it seems that  @jiangmaniu does not use Auto Rename Tage. @HenryC-3 your issue looks more like #132

**Comment by amythos** [comment]
hi, @alexanderniebuhr i have disable all my vscode extensions except WindiCSS IntelliSense , and then reenable those extensions like @jiangmaniu does. and now WindiCSS IntelliSense works for me.

and i found the problem is the context of windi.config.ts from [windicss configuration example](https://windicss.org/guide/configuration.html), maybe WindiCSS IntelliSense got some error when loading the default config, but without log the errors.

![Snip20210527_2](https://user-images.githubusercontent.com/959296/119787400-bf496f00-bf03-11eb-9d4b-00a94f60d554.png)

![Snip20210527_4](https://user-images.githubusercontent.com/959296/119788341-a2fa0200-bf04-11eb-9a70-9c45cf1647c7.png)

**Comment by alteredorange** [comment]
I had a similar problem with the plugin. One day intelli-sense stopped working and I couldn't for the life of me figure it out. As far as I can tell, the plugin prefers a `.js` file.  I had `.ts` and `.cjs` on different projects, and it was working, but then stopped.  Adding a blank `windi.config.js` fixed the problem immediately. Maybe the plugin prefers `.js` and has a hard time picking up other extensions?  Right now I have my actual config (`windi.config.cjs`), and a copy of that as a `.js` file so intellisense will work.

**Comment by leosin** [comment]
+1  in version 0.16.5

**Comment by ycs77** [comment]
@HenryC-3 Disable Auto Rename Tag plugin is work for me, thanks!

**Comment by voorjaar** [comment]
v0.18.3 should fix this. and also added more detailed log info.

**Comment by voorjaar** [comment]
found another bug, if enable auto rename tag, and require some plugin like typography windi plugin will crash.

**Comment by laozei6401** [comment]
Disable automatic renaming to take effect

**Comment by jbaubree** [comment]
In my case, i had same issue and i fixed it directly importing Windicss in my package.json. I think vite-plugin-windicss does not link well windicss for vs-code.
You can also try to export your windicss config file without defineConfig(), it will be better because vscode will not search anything from 'windicss'.

Hope that will help :)
